cmd/entrypoint/README.md

```diff
@@ -30,6 +30,12 @@ The following flags are available:
   same value as `{{stdout_path}}` so both streams are copied to the same
   file. However, there is no ordering guarantee on data copied from both
   streams.
+- `-enable_spire`: If set will enable signing of the results by SPIRE. Signing
+  results by SPIRE ensures that no process other than the current process can
+  tamper the results and go undetected.
+- `-spire_socket_path`: This flag makes sense only when enable_spire is set. 
+  When enable_spire is set, spire_socket_path is used to point to the
+  SPIRE agent socket for SPIFFE workload API.
```

Comment:
@jagathprakash you mentioned that the list here is incomplete - i wonder in the long run if it would make the most sense to avoid listing the individual commands here and just talk more generally about the features (and rely on the docs for the command itself vs trying to keep this in sync) - e.g. a list of features that the entrypoint binary supports such as signing with spire, etc. vs. specifics about these flags. In that case I could see there being a small section about SPIRE below. Anyway this is a good start in at least having some docs.